nixos/modules/services/networking/pppd.nix

```diff
@@ -0,0 +1,137 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  meta = {
+    maintainers = with maintainers; [ danderson ];
+  };
+
+  options = {
+    services.pppd = {
+      package = mkOption {
+        default = pkgs.ppp;
+        defaultText = "pkgs.ppp";
```

Comment:
Is this necessary, I thought the document system could cope with this

Reply:
I don't know, I was copying other nixos modules that do this with their pkg reference. I'm happy to remove it and see what happens when the docs go live?

Reply:
It can't hurt so leave it

Reply:
There is probably a way to locally build the docs but I'm not quite sure how if you wanted to check

Reply:
I'll look around tomorrow for a way to build the options search engine locally and check for this.